test/integration/model/find.test.js

```diff
@@ -121,7 +121,7 @@ describe(Support.getTestDialectTeaser('Model'), function() {
           where: ['specialkey = ?', 'awesome'],
           logging: function(sql) {
             test = true;
-            expect(sql).to.contain("WHERE specialkey = 'awesome'");
+            expect(sql).to.match(/WHERE specialkey = N?'awesome'/);
```

Comment:
The `?` should not be present in the final result, as evident from the existing test.